osquery/events/darwin/openbsm.cpp

```diff
@@ -65,7 +65,7 @@ Status OpenBSMEventPublisher::configureAuditPipe(FILE* au_pipe) {
     ev_classes.push_back("ad");
   }
 
-  if (true == FLAGS_audit_allow_fim_events)
+  if (true == FLAGS_audit_allow_fim_events) {
```

Comment:
I dont have macos build system ready and I have to cut out the piece of code which was relevant for contribution so thats why I am fixing build issues here. Sorry for trouble.